kart/point_cloud/import_.py

```diff
@@ -0,0 +1,147 @@
+from glob import glob
+import json
+from os.path import expanduser
+import sys
+
+import click
+
+from kart.exceptions import (
+    InvalidOperation,
+    NotFound,
+    NO_IMPORT_SOURCE,
+    INVALID_FILE_FORMAT,
+)
+from kart.output_util import format_wkt_for_output
+
+
+@click.command("point-cloud-import", hidden=True)
+@click.pass_context
+@click.argument("sources", metavar="SOURCES", nargs=-1, required=True)
+def point_cloud_import(ctx, sources):
+    """
+    Experimental command for importing point cloud datasets. Work-in-progress.
+    Will eventually be merged with the main `import` command.
+
+    SOURCES should be one or more LAZ or LAS files (or wildcards that match multiple LAZ or LAS files).
+    """
+    import pdal
+
+    all_source_paths = []
+    for source in sources:
+        source_paths = list(glob(expanduser(source)))
+        if not source_paths:
+            raise NotFound(NO_IMPORT_SOURCE, f"No data found at {source}")
```

Comment:
```suggestion
            raise NotFound(f"No data found at {source}", exit_code=NO_IMPORT_SOURCE)
```

can you make BaseException's kwargs keyword-only? That would avoid this problem to some extent

Reply:
Done and done